rule-tester-and-docs.js

```diff
@@ -73,13 +73,13 @@ class RuleTesterAndDocs extends RuleTester {
 
 			output += '## Rule details\n\n';
 
-			output += '✗ The following patterns are considered errors:\n' +
-				buildRuleDetails( tests.invalid, '✗' );
+			output += '❌ The following patterns are considered errors:\n' +
+				buildRuleDetails( tests.invalid, '❌' );
 
 			output += '\n';
 
-			output += '✓ The following patterns are not considered errors:\n' +
-				buildRuleDetails( tests.valid, '✓' );
+			output += '✔️ The following patterns are not considered errors:\n' +
```

Comment:
Maybe use ✅ instead?

Reply:
The other one renders as a green tick in GitHub, complementing the red 'x'

Reply:
> The other one renders as a green tick in GitHub, complementing the red 'x'

Only on your browser/OS, not on mine:

![image](https://user-images.githubusercontent.com/881572/65899010-a58b0980-e367-11e9-960f-6eb983cbd85e.png)

Reply:
I mean the one that is in the commit renders green (although it won't in the diff). GitHub (where most people will read the docs?) replaces emoji with images, so it shouldn't be browser-dependent. Here's how the current patch looks for me on the site:

![image](https://user-images.githubusercontent.com/180672/65899744-46031d00-e3ac-11e9-9e99-8e21e1a52654.png)

Reply:
Oh, yeah, if I look at the "rich diff" it does indeed look better.